changelogs/fragments/119-doc-updates.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - Fixed alignment, formatting, and typos in module documentation.
+  - Added explicit license to private module util.
+  - Updated out of date runtime version info.
```

These should link to either the PR itself, or the issue(s) the PR resolves, or both; also try to follow the recommended format, see: https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format

```suggestion
  - most modules - fixed alignment, formatting, and typos in module documentation (https://github.com/lowlydba/lowlydba.sqlserver/pull/119).
  - _SqlServerUtils module_util - added explicit license to private module util (https://github.com/lowlydba/lowlydba.sqlserver/pull/119).
  - meta/runtime.yml - updated out of date runtime version info (https://github.com/lowlydba/lowlydba.sqlserver/pull/119).
```